### PR TITLE
Added Korean explanation of Heap Sort + Fix typo

### PR DESCRIPTION
**Pull request by woo-jk**
A Korean description of heap sort was added, and there was a typo in the English description of heap sort where heap sort was a selection sort, so it was corrected. I want to be added. Thank you.

### CODE REVIEW

File: ko/정렬/버블정렬.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+[버블정렬 알고리즘에 대한 영상 설명](https://www.youtube.com/watch?v=Jdtq5uKz-w4)
+
+#### 그 외
+
+버블 정렬은 싱킹 정렬이라고도 한다.
+
+#### 애니메이션 설명
+
+- [Tute Board](https://boardhub.github.io/tute/?wd=bubbleSortAlgo2)
```
Comment (Panquesito7) [inline ko/정렬/버블정렬.md:104]: Please translate this string in Korean (not the link, the text itself).

File: ko/정렬/힙정렬.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+0번 인덱스의 힙 절차 적용:
+       10(0)
+       /  \
+    5(1)  3(2)
+   /   \
+4(3)    1(4)
+힙 절차는 재귀적으로 호출하여 하향식 방식으로 힙을 빌드한다.
+```
+
+![힙 이미지](https://upload.wikimedia.org/wikipedia/commons/1/1b/Sorting_heapsort_anim.gif "Heap Sort")
```
Comment (Panquesito7) [inline ko/정렬/힙정렬.md:54]:
```suggestion
![힙 이미지](https://upload.wikimedia.org/wikipedia/commons/1/1b/Sorting_heapsort_anim.gif)
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please resolve the merge conflicts [**here**](https://github.com/TheAlgorithms/Algorithms-Explanation/pull/158/conflicts). Let us know if you need any help! :)

**Comment by woo-jk** [comment]
I have resolved both requests and conflicts. :)

File: ko/정렬/버블정렬.md [hunk 3]
```diff
@@ -101,4 +101,5 @@
 
 #### 애니메이션 설명
 
-- [Tute Board](https://boardhub.github.io/tute/?wd=bubbleSortAlgo2)
+- [튜트 보드](https://boardhub.github.io/tute/?wd=bubbleSortAlgo2)
+
```
Comment (Panquesito7) [inline ko/정렬/버블정렬.md:105]:
```suggestion
```

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**